lib/handleRequest.js

```diff
@@ -54,9 +50,9 @@ function handleRequest (req, res, params, context) {
 }
 
 function handler (reply) {
-  var valid = validateSchema(reply.context, reply.request)
-  if (valid !== true) {
-    reply.code(400).send(wrapValidationError(valid))
+  var result = validateSchema(reply.context, reply.request)
+  if (result instanceof Error) {
```

Comment:
Can you run a benchmark with some validation? instanceof is very expensive. Can you restore it with a boolean?